desktop/cmp/panel/impl/PanelRefreshContextModel.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2019 Extremely Heavy Industries Inc.
+ */
+import {RefreshContextModel} from '@xh/hoist/core/refresh';
+import {TabRefreshMode} from '@xh/hoist/enums';
+import {loadAllAsync} from '@xh/hoist/core';
+
+/**
+ * @private
+ */
+@RefreshContextModel
+export class PanelRefreshContextModel {
```

Comment:
As discussed on Slack, please remove this new class from the branch - but great that it is on its way! 😄